bot.js

```diff
@@ -30,6 +32,48 @@ client.once('ready', () => {
     logMessage('Ready!');
 });
 
+async function fetchGlowStats() {
+    try {
+        const priceResponse = await axios.get('https://www.glowstats.xyz/api/tokenPrice');
+        const holdersResponse = await axios.get('https://www.glowstats.xyz/api/tokenHolders');
+        const farmsResponse = await axios.get(`https://www.glowstats.xyz/api/farmCount`);
+        const outputResponse = await axios.get('https://www.glowstats.xyz/api/weeklyOutput');
+
+        const priceData = priceResponse.data;
+        const holdersData = holdersResponse.data;
+        const farmsData = farmsResponse.data;
+        const outputData = outputResponse.data;
+
+        return {
+            uniswapPrice: priceData.tokenPriceUniswap,
+            contractPrice: priceData.tokenPriceContract / 10000,
+            tokenHolders: holdersData.tokenHolderCount,
+            numberOfFarms: farmsData[farmsData.length-1].count,
+            powerOutput: outputData[outputData.length - 1].output / 1000000
+        };
+    } catch (error) {
+        console.error('Error fetching glow stats:', error);
```

Comment:
you can log this using the logMessage function which will log it to a file and to the console

Reply:
We could extend that log function a bit to log 'console.error' too since it only logs 'info' at the moment